src/components/UI/Overlay/Overlay.stories.mdx

```diff
@@ -1,6 +1,12 @@
-import { useState } from "react";
```

Comment:
the doc panel is completely broken (I dont care that much)

However I think it would be nice to show what happens when we click on the element of the overlay:
=> clicking on the cross would display it in the action panel of the storybook
=> same for esc